src/services/signatureHelp.ts

```diff
@@ -485,8 +484,7 @@ function getArgumentIndex(argumentsList: Node, node: Node, checker: TypeChecker)
     // The list we got back can include commas.  In the presence of errors it may
     // also just have nodes without commas.  For example "Foo(a b c)" will have 3
     // args without commas. We want to find what index we're at.  So we count
-    // forward until we hit ourselves, only incrementing the index if it isn't a
-    // comma.
+    // forward until we hit ourselves.

```

Comment:
I removed this part of the comment because I adjusted the behavior (see `tests/cases/fourslash/signatureHelpSkippedArgs1.ts`). I find the new behavior better and it was easier for me to rewrite those loops while accommodating for that test case.